### PR TITLE
Enterprise

### PR DESCRIPTION
**Pull request by nmattia**
Some tweaks for the enterprise offering

### CODE REVIEW

File: src/blog/the-unbreakable-slides-for-your-corporate-communication.md [hunk 1]
```diff
@@ -80,39 +78,39 @@ Engage with your audience in real time by running polls within your slides. Get
...
 
-Write notes for your slides. If you wish, it would even be possible to publish these within your slides.
+Write notes for your slides. Embed them in your presentation when needed.
 
 *****
 
 #### Open Source
 
-DeckDeckGo is open source. All the code of our applications and components are available on [Github](http://github.com/deckgo/deckdeckgo).
+DeckDeckGo is open source. All the code of our applications and components is available on [Github](http://github.com/deckgo/deckdeckgo).
```
Comment (sacwrites) [inline src/blog/the-unbreakable-slides-for-your-corporate-communication.md:99]:
```suggestion
DeckDeckGo is open source. All of the code from our applications and components is available on [Github](http://github.com/deckgo/deckdeckgo).
```

File: src/blog/the-unbreakable-slides-for-your-corporate-communication.md [hunk 2]
```diff
@@ -80,39 +78,39 @@ Engage with your audience in real time by running polls within your slides. Get
...
 
-We encourage enterprise to adopt this approach but we do understand if you prefer to keep your corporate templates private.
+We encourage companies to adopt this approach but we do understand if you prefer to keep your corporate templates private.
```
Comment (sacwrites) [inline src/blog/the-unbreakable-slides-for-your-corporate-communication.md:101]:
```suggestion
We encourage companies to adopt this approach, but we do understand if you prefer to keep your corporate templates private.
```

File: src/blog/the-unbreakable-slides-for-your-corporate-communication.md [hunk 3]
```diff
@@ -80,39 +78,39 @@ Engage with your audience in real time by running polls within your slides. Get
...
 
 *****
 
 #### Developer Kit
 
-All public features used by our editor are available as a developer kit therefore, if you rather like to prepare your presentation with HTML or Markdown, go for it `npm init deckeckgo` .
+All public features used by our editor are available as a developer kit. Therefore, if you would rather like to prepare your presentation with HTML or Markdown, go for it with `npm init deckeckgo`.
```
Comment (sacwrites) [inline src/blog/the-unbreakable-slides-for-your-corporate-communication.md:107]:
```suggestion
All public features used by our editor are available as a developer kit. Therefore, if you would rather prepare your presentation with HTML or Markdown, go for it by using `npm init deckeckgo`.
```

File: src/blog/the-unbreakable-slides-for-your-corporate-communication.md [hunk 4]
```diff
@@ -80,39 +78,39 @@ Engage with your audience in real time by running polls within your slides. Get
...
 
 *****
 
 ### Get In Touch
 
-I hope the above introduction and having given a try to [DeckDeckGo](https://deckdeckgo.com) made you curious about our solution for enterprise. Get in touch for a tailored discussion about your needs, send us an [email](mailto:hello@deckdeckgo.com).
+I hope the above introduction and having given [DeckDeckGo](https://deckdeckgo.com) a try made you curious about our solution for enterprise. Get in touch for a tailored discussion about your needs by sending us an [email](mailto:hello@deckdeckgo.com).
```
Comment (sacwrites) [inline src/blog/the-unbreakable-slides-for-your-corporate-communication.md:113]:
```suggestion
I hope the above introduction (and giving [DeckDeckGo](https://deckdeckgo.com) a try for yourself) made you curious about our solutions for enterprise clients. Get in touch for a tailored discussion about your needs by sending us an [email](mailto:hello@deckdeckgo.com).
```

File: src/blog/the-unbreakable-slides-for-your-corporate-communication.md [hunk 5]
```diff
@@ -14,53 +14,51 @@ canonical: "https://medium.com/@david.dalbusco/app-shortcuts-and-maskable-icons-
...
 
 *****
 
 #### Custom Domain, In Housing and Share Privately
 
-You can publish your decks under your own domain name. It is also possible to create private links to prevent access to your published content.
+You can publish your decks under your own domain name. It is also possible to create authenticated links for private content.
 
-If you rather like to integrate DeckDeckGo in your infrastructure and private network rather than using our SASS platform, we are open to the ideas too.
+FInally, if you would rather integrate DeckDeckGo in your infrastructure and private network instead of using our SASS platform, we are here to help.
```
[the remark this answers is not in the document]
Comment (nmattia) [reply src/blog/the-unbreakable-slides-for-your-corporate-communication.md:55]: ah, yes indeed!

File: src/blog/the-unbreakable-slides-for-your-corporate-communication.md [hunk 6]
```diff
@@ -14,53 +14,51 @@ canonical: "https://medium.com/@david.dalbusco/app-shortcuts-and-maskable-icons-
...
+It's astonishing how much the look and feel of presentations vary, even within the same company. Templates, formatting, color palettes, fonts, logo – you name it, they are never the same across two decks. Presentations are rarely consistent, even when companies invest time and effort in brand identity and design guidelines.
 
-On the other hand, when we created [DeckDeckGo](https://deckdeckgo.com), a web editor for presentations, we designed it to be powered by self-contained templates that may, or may not, allow a certain number of features, with no other limitation that what the web can do.
-
-Base on these two observations, we are thrilled to unveil today our offering for [enterprise](https://deckdeckgo.com/enterprise) customers.
+Today we are thrilled to announce that [DeckDeckGo](https://deckdeckgo.com) is receiving support for templates for our [enterprise](https://deckdeckgo.com/enterprise) customers. Create a template once and all presentations will then use the same.
```
Comment (peterpeterparker) [inline src/blog/the-unbreakable-slides-for-your-corporate-communication.md:19]: for [enterprise](https://deckdeckgo.com/enterprise) customers

(I'll remove "our")

File: src/blog/the-unbreakable-slides-for-your-corporate-communication.md [hunk 7]
```diff
@@ -14,53 +14,51 @@ canonical: "https://medium.com/@david.dalbusco/app-shortcuts-and-maskable-icons-
...
 
-If you rather like to integrate DeckDeckGo in your infrastructure and private network rather than using our SASS platform, we are open to the ideas too.
+Finally, if you prefer to integrate DeckDeckGo into your infrastructure and private network instead of using our SaaS platform, we are here to help.
 
 *****
 
 ### And So Much More
 
-These above main features are the one we think are specific to business but DeckDeckGo is so much more than that.
+That rounds up the new features for our enterprise customers – but we're also releasing a ton of new things for private users!
```
Comment (peterpeterparker) [inline src/blog/the-unbreakable-slides-for-your-corporate-communication.md:61]: That rounds up the new features for our enterprise customers – but we have already released a ton of things for any users!

**Pull request merged**